util/src/main/java/edu/asu/diging/gilesecosystem/util/files/impl/FileStorageManager.java

```diff
@@ -237,6 +237,38 @@ public String getBaseDirectory() {
     public String getBaseDirectoryWithFiletype() {
         return baseDirectory + File.separator + fileTypeFolder;
     }
+    
+    @Override
+    public byte[] getExtractedFileContent(String username, String uploadId, String documentId, int pageNr, String filename) {
```

Comment:
what is this method for? Is this to get files from the script that tardis executes? If so, that logic should not be here. This class is a generic class that should only have methods that could be used anywhere.